pipelines/tekton/gitops-update-pipeline/example-pipelineruns/gitops-update-pipelinerun-bike-rentals.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  labels:
+    tekton.dev/pipeline: gitops-update-pipeline
+  generateName: gitops-update-pipeline-bike-rentals-
+spec:
```

Comment:
I noticed the error below when I deployed this to a fresh cluster.  I didn't have time to debug but there was no issue when creating the tensorflow-housing PipelineRun.  

```
Error from server (BadRequest): error when creating "pipelines/tekton/gitops-update-pipeline/example-pipelineruns/gitops-update-pipelinerun-bike-rentals.yaml": admission webhook "webhook.pipeline.te
kton.dev" denied the request: mutation failed: cannot decode incoming new object: json: unknown field "spec"
```

Reply:
I think I see the problem...it's an indentation error in the volumeclaimtemplate on line 28. Will fix ASAP, thanks

Reply:
This is fixed now, and I've verified that the fix works on the older cluster. Sorry for the noise!